### ISSUE TITLE
PDF and other links

### ISSUE THREAD

**Issue by tbl73** [VictorianResearchWeb/TheCurranIndex#11]
The original Curran index website includes some hyperlinks to archive.org and to various PDFs - we need to determine how those should be handled.  Are the links included in the database already, or were they manually added?  Should the PDFs be displayed, and if so, what's the best method?

**Comment by gsimons1946** [comment]
The hyperlinks to archive.org were an experiment of linking to the text of articles. We only did it for a few articles; at this point I think they can be ignored.

The pdfs are of two types: (1) some are to article files which were simply too big to manually convert to html. This information is in the MYSQL database, so the linkages on the CI website can be ignored. (2) linkages to release notes for various updates were shown as pdf files. These files are now entered into the CI database under the Essays table. At a relatively low priority it would be good if a user could read these files.

Bottom line: no need to display these hyperlinks in the new website

**Comment by tbl73** [comment]
Great - I'll close this issue, then.  The essays table is already broken out into another issue.
